cmd/terramate/cli/cli.go

```diff
@@ -259,31 +260,46 @@ func Exec(
 ) {
 	configureLogging(defaultLogLevel, defaultLogFmt, defaultLogDest,
 		stdout, stderr)
-	c := newCLI(version, args, stdin, stdout, stderr)
+	c := newCLI(
+		version,
+		args,
+		stdin,
+		stdout,
+		stderr,
+		printer.NewPrinter(stdout),
```

Comment:
AFAICS there's no need to configure the printers, they can be instantiated in the `newCLI()` constructor, no?

Reply:
Yes that makes sense 👍🏾